tests/integration/devtools/test_devtools_error_handling.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: UTF-8 -*-
+import dash_html_components as html
+import dash_core_components as dcc
+import dash
+from dash.dependencies import Input, Output
+from dash.exceptions import PreventUpdate
+
+
+def test_dveh001_python_errors(dash_duo):
```

Comment:
@alexcjohnson  the tcid is added, it's more precise to communicate the test case with a unique id. and in case of test selection, mostly of time you can play with `pytest -k tcid`. 

the current convention is `mmffddd`  abbr for module name and file and three digits,  it also helps to navigate to the test case in IDE if you have the tcid